Sofia-Rojas/tarea1/index.html

```diff
@@ -0,0 +1,29 @@
+<html>
+    <head>
+        <title> Magazine Cover 2
+        </title>        
+        <link rel="stylesheet" href="C:\Users\sofia.rojas\Documents\VSC\Tarea 1\css\Tarea 1 intento 2 css.css">
```

Comment:
@sofia-rojas , please use relative paths, this only works in your computer, also change the name for CSS file it should not have spaces within the filename.

```suggestion
        <link rel="stylesheet" href="css\styles.css">
```